chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -240,8 +262,133 @@ Notes: Script here
 
 ---
 
-Using `pandas` to plot groupby objects is very limited and is not possible for many other plot types such as scatter plots.      
-If you wish to learn more advanced visualization and Python plotting package **Altair**, come back soon and visit our in development course **DSCI-031 Exploratory Data Visualization**. 
+Let's go through the steps that were needed to make the plot in the previous slide. 
+
+- We created a groupby object and calculated the mean for each column in the resulting dataframe.       
+- Next, we took the single column we are interested in using `.loc[]`.
+- Since grouping by made `mfr` the new index, we had to use `reset_index(inplace=True)` to make `mfr` a column again.
```

Comment:
since `.groupby()` made `mfr` the new index ... 

ALSO NO INPLACE!!! SAVE AS A NEW OBJECT!!!